warehouse/query-core/src/main/java/datawave/query/function/IndexOnlyKeyToDocumentData.java

```diff
@@ -38,9 +38,9 @@
  * Fetches index-only tf key/values and outputs them as "standard" field key/value pairs
  */
 public class IndexOnlyKeyToDocumentData extends KeyToDocumentData implements Iterator<Entry<DocumentData,Document>> {
-    private static final Collection<ByteSequence> COLUMN_FAMILIES = Lists.<ByteSequence> newArrayList(new ArrayByteSequence("d"));
+    private static final Collection<ByteSequence> COLUMN_FAMILIES = Lists.newArrayList(new ArrayByteSequence("d"));
```

Comment:
is the d constant defined somewhere else it can be referenced here?

Reply:
I checked but didn't find anything. I would re-use the one from KeyToDocumentData, but those column families include the TF column and the IndexOnly variant explicitly excludes that.